x-pack/test/lists_api_integration/security_and_spaces/tests/read_lists.ts

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect';
+
+import { FtrProviderContext } from '../../common/ftr_provider_context';
+import { LIST_URL } from '../../../../plugins/lists/common/constants';
+
+import {
+  getCreateMinimalListSchemaMock,
+  getCreateMinimalListSchemaMockWithoutId,
+} from '../../../../plugins/lists/common/schemas/request/create_list_schema.mock';
+import { createListsIndex, deleteListsIndex, removeServerGeneratedProperties } from '../../utils';
+import { getListResponseMockWithoutAutoGeneratedValues } from '../../../../plugins/lists/common/schemas/response/list_schema.mock';
+
+// eslint-disable-next-line import/no-default-export
+export default ({ getService }: FtrProviderContext) => {
+  const supertest = getService('supertest');
+
+  describe('read_lists', () => {
+    describe('reading lists', () => {
+      beforeEach(async () => {
+        await createListsIndex(supertest);
+      });
+
+      afterEach(async () => {
+        await deleteListsIndex(supertest);
+      });
+
+      it('should be able to read a single list using id', async () => {
+        // create a simple list to read
+        await supertest
```

Comment:
Is the preference to do this, rather than using fixtures? Ideally we'd keep the scope of this test small and just test reads (this is, in effect, testing both create and read), but on the other hand... maintenance of fixtures can be "not fun." Just curious if there's ever been any discussion as to what the standard should be ... I see a good mix of both across the different plugins. FWIW, we used a bit of a mix for the endpoint artifact packaging code, but any setup calls were done in `before*`.

Reply:
There hasn't been much discussion, I haven't used fixtures before other than other people's.

So far this has been simple to maintain and when things blow up it hasn't been hard to trace it down to the right spot. I'm not against learning fixtures though. Seems like it might be more maintenance like the issues with loading saved objects from esArchiver has been a big pain point.

Overall, I have been maintaining the detection engine tests using mostly REST setup, tear downs, and the only maintenance has been two suite skips due to networking issues we solved by using retry loops (which would still have happened with fixtures or dummy data).

Reply:
I agree it can make maintenance a little more painful. Carry on. :)